marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -159,15 +159,11 @@ class Flow {
       query = sourceQuery ? xdmp.eval(sourceQuery) : null;
     }
 
-    if (stepDefinition.name === 'default-merging' && stepDefinition.type === 'merging' && uris) {
```

Comment:
This seems really good that we're removing this, as ideally the flow framework has no knowledge of concrete step types. I'm missing what changes in merging.sjs make this possible though?

Reply:
Now the merging step will process an entire match summary at a time instead of a single document referenced in the match summary.

Reply:
Got it - the inclusion of "thisMatchSummaryURI" as a variable name in merging.sjs makes that clear too.